docs/usage.rst

```diff
@@ -96,27 +97,45 @@ you can pass ``HTML`` to render whatever you want.
 
 .. code-block:: python
 
-    from beam.layouts import VirtualField, HTML
-
-    class ExampleViewSet(beam.ViewSet):
-        model = Example
-        fields = ["name", "phone"]
-        layout = [[
-            [
-            "name",
-            VirtualField(
-                "phone",
-                lambda obj: mark_safe(obj.phone_as_link),
-                verbose_name=_("phone"))
-            ],
-            [
-            HTML("<h1>Some HTML</h1>")
-            ]
-        ]]
+    # books/models.py
+    class Book(models.Model):
+        title = models.CharField(max_length=255)
+        author = models.ForeignKey("Author", on_delete=models.CASCADE, related_name="books")
+        publication_date = models.DateField()
+        price = models.DecimalField(max_digits=5, decimal_places=2)
 
+        # add this field
+        @property
+        def isbn_search(self):
```

Comment:
This sort of makes it look like you need to have that property here.
I suggest we just construct the link the viewset where we access `obj.isbn`.